templates/web/base/admin/bodies/contact-form.html

```diff
@@ -74,6 +74,11 @@
         <span class='form-hint'>[% loc('Use this where you do not want problem reporters to be able to reopen their fixed or closed reports when leaving an update.') %]</span>
     </p>
 
+    <p class="form-check">
```

Comment:
It might be an idea to hide or disable this field unless the body's cobrand has `anonymous_account` set, otherwise clicking the 'report anonymously' button when making a report results in a "please enter your email" error.

Reply:
yes, good catch.